src/main/java/com/cloudbees/jenkins/GitHubCommitNotifier.java

```diff
@@ -34,9 +42,16 @@
  */
 public class GitHubCommitNotifier extends Notifier {
 
+    private ExpandableMessage statusMessage = new ExpandableMessage("");
```

Comment:
User should see real configuration that would be applied, if it 4 fields, then 4.